templates/unbound.conf.erb

```diff
@@ -182,3 +182,7 @@ server:
 <% @custom_server_conf.each do |c| -%>
   <%= c %>
 <% end -%>
+<% if @python -%>
+python:
```

Comment:
Is this the correct syntax?  Looking in the manpager makes it seem like `python-script` is the top level key here.